scripts/data/v3/infrastructure-damaged.ts

```diff
@@ -0,0 +1,147 @@
+import { ApiResource } from "../../../types/api.types";
+import { writeJson } from "../../utils/fs";
+import { SheetTab, fetchGoogleSheet } from "../../utils/gsheets";
+import {
+  formatDailiesJson,
+  validateDailiesJson,
+} from "../common/casualties-daily";
+
+const jsonFileName = "infrastructure-damaged.json";
+
+const columnFilter = new Set([
+  "report_date",
+  "civic_buildings.destroyed",
+  "ext_civic_buildings.destroyed",
```

Comment:
This comment is less important than the others, feel free to ignore if you don't think it would improve it.

---

An alternative to this separated nesting that might make more sense could be:

```json
{
  "civic_buildings": {
    "destroyed": 1,
    "ext_destroyed": 1
  }
}
```

In other words, not having `ext_civic_buildings` as a separate sibling object.

Then your parent fields could be directly related to the distinct types of infrastructure:
* civic_buildings
* educational_buildings
* places_of_worship (maybe group mosques and churches? ie: `places_of_worship.mosques_destroyed`)
* residential

The non-ext fields would still be optional based on the reporting day, but since the two values are very related for a given infrastructure type I think having the nesting this way may make more sense. The alternative is to not nest at all, I don't think it's required. I don't see us capturing any more values in those nested objects so having them all separate may not be necessary and just harder to follow as a user.